modules/nf-core/rgi/main/main.nf

```diff
@@ -60,7 +60,16 @@ process RGI_MAIN {
         --input_sequence $fasta
 
     mkdir temp/
-    mv *.xml *.fsa *.{nhr,nin,nsq} *.draft *.potentialGenes *{variant,rrna,protein,predictedGenes,overexpression,homolog}.json temp/
+    mv *.xml *.fsa *.{nhr,nin,nsq} *.draft *.potentialGenes temp/
+
+    ## These files not always generated, so only move if they exist
+    ## Taken from https://stackoverflow.com/a/54655320 (RIP) by @TSJNachos117
+    if [ "\$(ls -A | grep -i \\variant.json\\\$)" ]; then mv *variant.json temp/; fi
```

Comment:
It'll do provided you align the semi-colons.

Reply:
done. Feels weird doing harshil align on bash but first time for anything